### PR TITLE
336 - Projects-with-complex-description-load-slow

### PR DESCRIPTION
**Pull request by waltersajtos**
## Description

If a project has a really long description it loads really slow, this pull request will replace the quill-view with a div that will inject the html directly into the element. This will improve the render time significantly 

## Type of change

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [ ] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a project with this github link: https://github.com/quozd/awesome-dotnet
2. The project should load in fast

## #336 

Closes: #336

### CODE REVIEW

**Comment by Brend-Smits** [comment]
So what was the issue and what is your solution? This is what the description is used for. Can you explain and edit the PR please?

File: src/app/utils/safeHtml.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({ name: 'safeHtml'})
+export class SafeHtmlPipe implements PipeTransform  {
+  constructor(private sanitized: DomSanitizer) {}
+  transform(value) {
+    return this.sanitized.bypassSecurityTrustHtml(value);
```
Comment (Brend-Smits) [inline src/app/utils/safeHtml.pipe.ts:8]: Why bypass the sanitization and not just sanitize it using the following code?
```suggestion
    return this.sanitized.sanitize(1, value);
```
I understand we already do some sanitization on the backend but does it hurt to do some on the frontend as well?

File: src/app/utils/safeHtml.pipe.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Pipe({ name: 'safeHtml'})
+export class SafeHtmlPipe implements PipeTransform  {
```
Comment (Brend-Smits) [inline src/app/utils/safeHtml.pipe.ts:5]: Missing documentation as to why this class exists. Also missing the license header :) Check other util files for reference.

**Review by Brend-Smits — CHANGES_REQUESTED** [review]
It is indeed significantly faster. In the future, we should look into loading the innerhtml part async and show like a spinning icon for the description part until it's ready.

File: src/app/utils/safeHtml.pipe.ts (hunk 1 above)
Comment (waltersajtos) [reply src/app/utils/safeHtml.pipe.ts:8]: I tested it and I thought it messed up the styling of the `build status icons` but apparently that was caused by something else.

**Review by Brend-Smits — APPROVED** [review]
LGTM! Thanks :)

**Review by Ruby77 — APPROVED** [review]
Looks good 👍

**Pull request merged**